front/src/utils/date.ts

```diff
@@ -18,17 +18,40 @@ export function formatIsoDate(date: Date) {
   return date.toISOString().substring(0, 10);
 }
 
-export function dateTimeFormatting(date: Date, withoutTime: boolean = false) {
+export function dateTimeFormatting(dateInput: Date | string, withoutTime = false): string {
```

Comment:
Hm, not a fan of adding more workarounds in this function. But why do we need this in the first place? Is the `last_modification` date string missing the timezone?

Reply:
Right, but why is that?